src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -433,6 +440,11 @@ public async Task RemoveAsync(ContentItem contentItem)
                     x.ContentItemId == contentItem.ContentItemId &&
                     (x.Published || x.Latest)).ListAsync();
 
+            if (activeVersions.Any())
```

Comment:
Shouldn't it just `return` here, as nothing needs to be done?

Reply:
Ok let's do that instead. Could be annoying to log it for those using javascript type of UI.